packages/admin/src/pages/scheme/[schemeId]/advert/[advertId]/preview/components/PreviewSearchResultCard.tsx

```diff
@@ -137,13 +129,7 @@ const PreviewSearchResultCard = ({ grant }: PreviewSearchResultCardProps) => {
               data-testid="ApplicationCloseDateValue"
             >
               {grant.grantApplicationCloseDate.length ? (
-                <Moment
-                  format="D MMMM YYYY, h:mma"
-                  tz="GMT"
-                  data-testid="closeDate"
-                >
-                  {adjustedCloseDate}
-                </Moment>
+                <time data-testid="closeDate">{adjustedCloseDate}</time>
```

Comment:
Are we preserving the formatting here? There were a couple tickets recently around this so just making sure we're not regressing that

Reply:
we should be, there are tests for it that I ensured were passing
```newDate.tz('GMT').format('D MMMM YYYY, h:mma');``` is the same formatting and timezone as was originally present, and <Moment/> is rendered as <time/> so everything should be exactly the same